customize.html

```diff
@@ -13,8 +13,7 @@ <h1>Content Database</h1>
 
 <p>
   Browse through a ton of content on the ContentDB website.<br>
-  In the next release, you'll even be able to install content without
-  leaving your Minetest client!
+  You'll also be able to install content right from your Minetest client!
```

Comment:
<i>will</i> be able?
this should be "are"

Reply:
Good catch. But having thought about it, I feel

`You can also install content right from your Minetest client!`

might sound even better (already committed, btw - sorry about that). What's your opinion on this?